### DISCUSSION TITLE
Regarding the ongoing o1 Waitlist

### DISCUSSION THREAD

**Discussion by Srajan04** [community/community#141237] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I had signed up for Github Models and o1 Models back on 26th September. I did get access to Github Models, but not for o1 Models. How much longer do I need to wait? Also I have tried and tested o1 Models on NAT but want to test and see if I can integrate them in my projects through Github Models API.

**Comment by MorelliniMatteo** [comment]
If you've already signed up for access to GitHub Models and o1 Models but only received access to GitHub Models, it's possible that the approval process for o1 Models is still ongoing. Access times can vary depending on several factors such as user demand or project requirements.

Here are a few steps you can take to follow up and potentially speed up the process:

Check Your Email: Ensure that you didn’t miss any communication from GitHub regarding the status of your o1 Models access. Sometimes approval notifications or additional steps are sent via email.

Reach Out to GitHub Support: You can directly contact GitHub support or the GitHub Models team through [GitHub's contact page](https://support.github.com/contact) or any support channels provided in your GitHub Models documentation. Ask for an update on your o1 Models access request.

Join the GitHub Discussions: If there’s a community forum or GitHub Discussions section related to GitHub Models, you can raise your question there. Other users or GitHub staff may provide insights or updates on the current status of o1 Models access.

GitHub API Access for o1 Models: Since you're interested in integrating o1 Models through the GitHub Models API, it’s worth mentioning that once your access is granted, you should be able to use the GitHub Models API to integrate o1 Models into your projects. Make sure to monitor GitHub’s documentation or any developer updates for relevant API endpoints and instructions.

While you’re waiting for approval, you can continue exploring the capabilities of GitHub Models and prepare your project for o1 Models integration once you gain access. If you've already tested o1 Models on NAT, that should give you a good head start.
